subprojects/mir-core.wrap

```diff
@@ -0,0 +1,4 @@
+[wrap-git]
+directory=mir-core
+url=https://github.com/libmir/mir-core.git
+revision=head
```

Comment:
A project that will use stdx-allocator would need to override `wrap` file anyway.